docker/docker-compose.1804.55.yaml

```diff
@@ -3,13 +3,16 @@ version: "3"
 services:
```

Comment:
Delete this file?

Reply:
+1 this still looks wrong so either has to go back to 18 or deleted

Reply:
Whoop, thanks will delete